cs/src/Management/IdGeneration.cs

```diff
@@ -0,0 +1,26 @@
+    private static readonly Random random = new Random();
+    /// <summary>
+    /// Generate valid tunnelIds
+    /// </summary>
+    /// <returns>string tunnel id</returns>
+    public static string GenerateTunnelId()
+    {
+        var sb = new StringBuilder();
+        sb.Append(adjectives[random.Next(adjectives.Length)]);
+        sb.Append("-");
+        sb.Append(nouns[random.Next(nouns.Length)]);
```

Comment:
Will that be too low number of variants? Should we have more parts in a random name? E.g. {adj}-{noun}-{preposition}-{adj}-{noun}

Reply:
We should use shorter words and try to generate more random combinations by increasing the number of words in the name.

Reply:
Updated to add random chars at the end of the id